packages/manager-components/src/components/typography/description/description.component.tsx

```diff
@@ -9,13 +9,15 @@ import { OsdsText } from '@ovhcloud/ods-components/react';
 
 export interface DescriptionProps {
   children?: string;
+  className?: string;
 }
 export const Description: React.FC<DescriptionProps> = ({
   children,
+  className,
 }: DescriptionProps) => {
   return (
     <OsdsText
-      className="block mb-6"
+      className={`block mb-6 ${className}`}
```

Comment:
```suggestion
      className={`block ${className}`}
```
remove hard-coded margin on component

Reply:
The purpose of this PR is not to refactor existing components and create breaking changes in apps that already use them. It is to update the dashboard template component.

We should discuss whether we include margin or not in our typography components and have the same approach in all of them in a separate ticket.

I agree with you though, I would rather not have margin included in this component.

Reply:
I've checked, this component is only used in `headers` and `dashboard` manager-components, so it would be ok to fix this on this PR (just have to update the `header` instance with a `mb-6` class)

But yes, i agree, it should be discussed with everyone.

Reply:
done since it doesn't impact any app